.travis.yml

```diff
@@ -37,8 +37,11 @@ _android: &_android
 
 matrix:
   include:
-    # one local test, without saucelabs
-    - env: PLATFORM=local
+    # Local test, without saucelabs (One current and one previous release)
+    - env: PLATFORM=local/ios-9.3
```

Comment:
Hm, that way we should also be able to run iOS _and_ Android locally, right?

Reply:
Yes. we can run both platforms locally with this setup.